node/src/database/db_initializer.rs

```diff
@@ -624,16 +608,35 @@ impl Debug for DbInitializationConfig {
 #[cfg(test)]
```

Comment:
Hmmm. It causes me pain making such a note but I believe this is breaking a pattern so we should move this whole module `test_utils` from here to a place like `node/src/database/test_utils` if one exists. Otherwise we ought to make one now and do it then.

Reply:
Okay, I did it.